pkgs/odri-controls.nix

```diff
@@ -0,0 +1,30 @@
+{
+
+  src-odri-control,
+
+  stdenv,
+  cmake,
+  eigen,
+  python312Packages,
+  python312,
+  yaml-cpp,
+  odri-masterboard-sdk,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "odri-control";
+  version = rosVersion ./package.xml;
```

Comment:
`rosVersion` needs the package.xml to be there locally, I guess I have to put this in the flake's overrideAttrs, but, in that case, what do I put here?